src/templates/static/legal/metatrader/order_execution.jsx

```diff
@@ -19,53 +19,52 @@ const OrderExecution = () => (
         <ul className='bullet'>
-            <li>{it.L('the specific instructions that the Company receives from the client;')}</li>
-            <li>{it.L('the market conditions of the financial instruments that client orders are related to, which includes the need for timely execution, availability of price improvement, liquidity of the market, size of client order, and the potential impact on total consideration.')}</li>
+            <li>{it.L('The specific instructions that the Company receives from the client')}</li>
+            <li>{it.L('The market conditions of the financial instruments that client orders are related to, which includes the need for timely execution, availability of price improvement, liquidity of the market, size of client order, and the potential impact on total consideration')}</li>
         </ul>
         <p>{it.L('The Company\'s determination of the relative importance of the execution factors may differ from the client\'s during certain circumstances, acting in the client\'s interest in accordance with the Company\'s obligation of best execution.')}</p>
 
         <h2 data-anchor='execution-venues'>{it.L('F. Execution venues')}</h2>
-        <p>{it.L('[_1] offers Forex and contracts for differences via the hybrid model, i.e. dealing on own account (dealing desk – B-book) or partially hedging client orders with the Company\'s liquidity providers (no-dealing desk – A-book).', it.website_name)}</p>
+        <p>{it.L('[_1] offers forex and contracts for differences via the hybrid model, i.e. dealing on own account (dealing desk – B-book) or partially hedging client orders with the Company\'s liquidity providers (no-dealing desk – A-book).', it.website_name)}</p>
         <p>{it.L('When the Company deals on own account (dealing desk), the Company acts as principal and not as an agent on the client\'s behalf and the Company therefore acts as the execution venue.')}</p>
         <p>{it.L('Whilst the Company acts as principal in respect of the client\'s orders, the Company also assesses the execution venues that the Company uses and upon which the Company places significant reliance to provide the best possible result for the execution of the client\'s orders. These venues typically consist of third-party investment firms, brokers, and/or liquidity providers.')}</p>
         <p>{it.L('[_1] assesses the choice of external execution venues regularly (at least once a year) with the intention of achieving a better result for the client based on the Company\'s best execution obligation.', it.website_name)}</p>
         <p>{it.L('When executing client orders using the A-book model, the Company directs the client\'s trade to one of its liquidity providers, namely:')}</p>
         <ul className='bullet'>
             <li>{it.L('GBE Brokers;')}</li>
-            <li>{it.L('Invast Financial Services Pty Ltd;')}</li>
```

Comment:
"Invast Financial Services Pty Ltd" (without the semicolon) should also appear on the list, after GBE Brokers.

Reply:
Okay